handlers/image.js

```diff
@@ -9,7 +9,7 @@ module.exports = browser => async (req, res) => {
   if (!state) res.status(400).send()
 
   try {
-    await page.goto(`http://localhost:${PORT}?state=${state}`)
+    await page.goto(`http://localhost:${PORT}?state=${encodeURIComponent(state)}`)
```

Comment:
We should be handling this properly in the [serialize/deserialize functions](https://github.com/dawnlabs/carbon/blob/master/lib/routing.js#L50) rather than inline on the handler

Reply:
Hey @mashihua can you move `encodeURIComponent` into the function @briandennis mentioned above? (: